mteb/tasks/Retrieval/law/AILAStatutesRetrieval.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class AILAStatutes(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="AILAStatutes",
+        description="This dataset is structured for the task of identifying the most relevant statutes for a given situation.",
+        reference="https://zenodo.org/records/4063986",
+        dataset={
+            "path": "mteb/AILA_statutes",
+            "revision": "ebfcd844eadd3d667efa3c57fc5c8c87f5c2867e",
+        },
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["test"],
+        eval_langs=["en"],
+        main_score="ndcg_at_10",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples=None,
+        avg_character_length=None,
```

Comment:
How to do the e filled out? Will your team do this for us?

Reply:
The [guide](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md) describes how to fill these out. Generally, we expect it when new datasets are added to MTEB as the ones adding the dataset are more knowledgeable about the dataset.

Reply:
Generally though here is a short guess:
        date=, # you best guess on date of when the data was generated (from, to) 
        form="written", # assumed 
        domains=["Legal", "Non-fiction"], # assumed
        task_subtypes=[],
        license=,  # needs be specified
        socioeconomic_status=["high"], # assumed but since it is law it is probably correct
        annotations_creators=, # required
        dialect=[], # assuming there are no dialects
        text_creation="found", # assumed
        bibtex_citation=, # if there is none just leave it as None otherwise please specify it 
        n_samples=, # check using python
        avg_character_length=, # check using python